### ISSUE TITLE
Add Example for Ad Connector

### ISSUE THREAD

**Issue by sivanagireddyb** [plus3it/terraform-aws-tardigrade-directory-service#92]
Value of ```connect_settings``` variable is set to list(string). But reading seems lis of objects ```connect_settings.value.customer_dns_ips``` . 

How will be variable format, for example, please? Below format throwing error. and ```vpc_settings``` option should not be present for the Ad connector it's only valid for simple and Microsoft AD.  How to handle this?
```
connect_settings = [
    customer_dns_ips  = ["A.B.C.D"]
    customer_username = "Admin"
    subnet_ids        = [aws_subnet.foo.id, aws_subnet.bar.id]
    vpc_id            = aws_vpc.main.id
]
```

**Comment by sivanagireddyb** [comment]
I am able to solve this by converting vpc_settings to dynamic for_each and vpc_settings as a list(object) variable.

**Comment by lorengordon** [comment]
@sivanagireddyb You're right, I think `connect_settings`` broke when we linted with tf 0.14, which changed it from `list` (implied `list(any)`) to `list(string)`...

And we hadn't tested with SimpleAD, so didn't catch the restriction on `vpc_settings`.

**Comment by lorengordon** [comment]
@sivanagireddyb Just opened a PR to address this, https://github.com/plus3it/terraform-aws-tardigrade-directory-service/pull/94...

**Comment by lorengordon** [comment]
@sivanagireddyb Support for AD Connector is included in [version 2.1.0](https://github.com/plus3it/terraform-aws-tardigrade-directory-service/releases/tag/2.1.0). There's an example of usage in the [`tests/ad_connector/`](https://github.com/plus3it/terraform-aws-tardigrade-directory-service/blob/29807d5233ce4c257409d1498aef095c1fbe6319/tests/ad_connector/main.tf#L5-L22) directory.
